Source/common/SNTKernelCommon.h

```diff
@@ -48,7 +48,8 @@ enum SantaDriverMethods {
 
 typedef enum {
   QUEUETYPE_DECISION,
-  QUEUETYPE_LOG
+  QUEUETYPE_LOG,
+  QUEUETYPE_COMPILER,
```

Comment:
As discussed we should be able to get rid of the compiler queue and using the decision queue. This only works if we are able to reduce the number of compiler messages sent over the queue by filtering based on known compiler pids in the kernel.